Fix Item bouncing + hand animation.

JohnButlerThird opened the pull request:
- Fixes witchery stairs/slabs having fucked up lighting.
- Backport item swinging animation when dropping item.
- Fixes item bouncing off blocks with complicated bounds.

JohnButlerThird committed:
Fixes witchery lighting

JohnButlerThird committed:
Added config to backport the throw items animation from modern.

JohnButlerThird committed:
Fix item bouncing off certain blocks

Co-Authored-By: Myask-sl <208960411+myask-sl@users.noreply.github.com>

JohnButlerThird committed:
remove unneded imports

JohnButlerThird committed:
Update MixinEntityItem.java

Co-Authored-By: Myask-sl <208960411+myask-sl@users.noreply.github.com>

JohnButlerThird committed:
Rename it by request

Co-Authored-By: Myask-sl <208960411+myask-sl@users.noreply.github.com>

JohnButlerThird committed:
Fix fence triggering unneeded right click animation.

JohnButlerThird committed:
Fix config not applying

JohnButlerThird committed:
i fixed the bugs LETS GO I HATE BUGS

JohnButlerThird committed:
Fix slab right click

JohnButlerThird committed:
remove client shit

JohnButlerThird committed:
Merge branch 'jss2a98aj:master' into master

JohnButlerThird committed:
Update MixinItemSlab.java

brandyyn commented:
@JohnButlerThird can you please remove the overwrite so this can be merged and a new release can be made?

JohnButlerThird commented:
> @JohnButlerThird can you please remove the overwrite so this can be merged and a new release can be made?
What overwrite ?

brandyyn commented:
> > @JohnButlerThird can you please remove the overwrite so this can be merged and a new release can be made?
> > What overwrite ?

all of them, you dont remember the crashes this PR was causing?

JohnButlerThird commented:
> > > @JohnButlerThird can you please remove the overwrite so this can be merged and a new release can be made?
> > > What overwrite ?
> 
> all of them, you dont remember the crashes this PR was causing?

I’m pretty sure I fixed it and moved to some inject. Tell me if you still get crashes
